src/plone/app/transmogrifier/uidupdater.py

```diff
@@ -9,6 +9,10 @@
 from plone.uuid.interfaces import IAttributeUUID
 from plone.uuid.interfaces import IMutableUUID
 from zope.interface import classProvides, implements
+import logging
+
+
+logger = logging.getLogger("plone.app.transmogrifier uidupdater")
```

Comment:
any reason not to use plain `plone.app.transmogrifier` logger?

Reply:
Yes, this is the same pattern the other blueprints are using - so its possible to see from the logger name what blueprint is used (see collective/transmogrifier/sections/constructor or plone/app/transmogrifier/reindexobject)